netbench/cdk/src/main/java/com/aws/NetbenchAutoApp.java

```diff
@@ -64,23 +64,36 @@ public static void main(final String[] args) {
             : ec2InstanceType.toLowerCase();
         
         // Stack instantiation
-        ReportStack reportStack = new ReportStack(app, "ReportStack", StackProps.builder()
-            .env(makeEnv(awsAccount, clientRegion))
+       ReportStack reportStack = new ReportStack(app, "ReportStack", StackProps.builder()
+            .env(makeEnv(awsAccount, serverRegion))
             .build());
 
-        ClientStack clientStack = new ClientStack(app, "ClientStack", ClientStackProps.builder()
```

Comment:
Can the Client and server stack be in the same stack? I think it could save you some future pain to avoid having circularly dependent stacks

Reply:
I don't think they're on the same stack, just created using the same class, but I could be wrong.

Reply:
To rephrase, I think you could reduce your number of stacks to the report stack and your "TestHarness" stack. You can utilize Constructs to create re-useable components in your "TestHarness" stack. 

From what I understand what you have so far should be deployed as one "Unit". You wouldn't necessarily want to deploy a client stack without a server stack (Or do you?)

Reply:
Oh I misunderstood your initial comment. I will look into a design like that and the cdk best practices guide.

Reply:
On second thought, I'm not sure putting the client and server on the same stack would be a good idea since they may need to be deployed to different AWS regions.

Reply:
Hmm that makes sense. I think you might run into another anti pattern https://github.com/aws/aws-cdk/issues/6242. Maybe the only thing you need to do is inject the IP address of the peer?